services/notifications/docker/boot.sh

```diff
@@ -47,26 +47,53 @@ APP_LOG_LEVEL=${LOGLEVEL:-${LOG_LEVEL:-${LOGLEVEL:-INFO}}}
 SERVER_LOG_LEVEL=$(echo "${APP_LOG_LEVEL}" | tr '[:upper:]' '[:lower:]')
 echo "$INFO" "Log-level app/server: $APP_LOG_LEVEL/$SERVER_LOG_LEVEL"
 
-if [ "${SC_BOOT_MODE}" = "debug" ]; then
-  reload_dir_packages=$(fdfind src /devel/packages --exec echo '--reload-dir {} ' | tr '\n' ' ')
 
-  exec sh -c "
-    cd services/notifications/src/simcore_service_notifications && \
-    python -Xfrozen_modules=off -m debugpy --listen 0.0.0.0:${NOTIFICATIONS_REMOTE_DEBUGGING_PORT} -m \
-    uvicorn \
-      --factory main:app_factory \
+if [ "${NOTIFICATIONS_WORKER_MODE:-}" = "true" ]; then
```

Comment:
THOUGHT: we should consider having some sort of shell script "library" with common functionality 

This is an example  https://github.com/just-containers/s6-overlay